InternxtDesktop/Services/BackupsService.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  BackupsService.swift
+//  InternxtDesktop
+//
+//  Created by Richard Ascanio on 1/8/24.
+//
+
+import Foundation
+import RealmSwift
+
+class BackupsService: ObservableObject {
+    @Published var foldernames: [FoldernameToBackup] = []
+    @Published var urls: [URL] = []
+    @Published var isBackupButtonEnabled: Bool = false
+
+    private func getRealm() -> Realm {
+        do {
+            return try Realm(fileURL: ActivityManager.realmURL)
+        } catch {
+            error.reportToSentry()
+            fatalError("Unable to open Realm")
+        }
+
+    }
+
+    func clean() throws {
+        foldernames = []
+        urls = []
+        let realm = getRealm()
+        try realm.write {
+            realm.deleteAll()
+        }
+    }
+
+    func addFoldernameToBackup(_ foldernameToBackup: FoldernameToBackup) {
+        guard let url = URL(string: foldernameToBackup.url) else {
+            return
```

Comment:
Don't return silently, specially in services, some alternatives:

- Pass an URL directly as parameter so we don't need to convert it
- Throw an error if we cannot generate an URL